jsapp/js/account/stripe.utils.ts

```diff
@@ -1,25 +1,32 @@
 import {when} from 'mobx';
 
-import {
-  getSubscription,
-} from 'js/account/stripe.api';
 import {ACTIVE_STRIPE_STATUSES} from 'js/constants';
 import envStore from 'js/envStore';
+import subscriptionStore from 'js/account/subscriptionStore';
 
 // check if the currently logged-in user has a paid subscription in an active status
 // promise returns a boolean, or `null` if Stripe is not active - we check for the existence of `stripe_public_key`
```

Comment:
Please turn this into `/**` comment, as it will be showing up in code editors as a helpful hint :)